hack/sync-api.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function usage() {
+    cat <<EOF
+ENV:
+    RAW: current repo the branch which contain github action ${{ github_ref }}
```

Comment:
RAW is not taken from an environment variable
`set +e; RAW=$(git branch -r --contains $REF 2>/dev/null);set -e`